### PR TITLE
Added even more words

### PR DESCRIPTION
**Pull request by eugene-chow**
@fonglh help me check if the escaping was done properly

### CODE REVIEW

File: words.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ var phrases = [
...
-  'Tamba',
+  'Tambah',
   'Tank',
   'No choice',
   'Means got problem',
   'There\'s the truth and then there\'s the brutal truth',
   'There is vision and then there is reality',
   'These people are not stooopid ok?',
-  '<cluck>',
+  '\<cluck\>',
```
Comment (fonglh) [inline words.js]: might have to use `&lt;` `&gt;` since it goes directly into the HTML

**Review by fonglh — CHANGES_REQUESTED** [review]

File: words.js (hunk 1 above)
Comment (jchiam) [reply words.js]: Yep I tried adding backslashes already. Didn't work

Comment (eugene-chow) [reply words.js]: changed to `&lt;` `&gt;`

**Review by fonglh — APPROVED** [review]

**Comment by fonglh** [comment]
Rebase and fix the merge conflict. Jonathan made changes in #4.

**Comment by eugene-chow** [comment]
oh man... how to rebase o.O

**Comment by jchiam** [comment]
`git rebase <upstream branch>`

Then resolve the merge conflicts

**Comment by eugene-chow** [comment]
ok i need help later... i'm not sure how this works

**Pull request merged**